src/lib/eliom_request.client.ml

```diff
@@ -140,6 +151,20 @@ let send
       | [] -> []
       | _ -> [ Eliom_common.tab_cookies_header_name,
                encode_header_value cookies ] in
+    let headers =
+      if Js.Unsafe.global##___eliom_use_cookie_substitutes_ <> Js.undefined then
+        (* Cookie substitutes are for iOS WKWebView *)
+        let host = match host with
+          | None -> "/substitutes" | Some h -> (h  ^ "/substitutes") in
+        let cookies =
+          Eliommod_cookies.get_cookies_to_send
+            ~in_local_storage:true (Some host) https path in
```

Comment:
You should just pass the initial `host` value here, without suffixing it with `/substitutes`